nixos/modules/config/nix-remote-build.nix

```diff
@@ -1,209 +1,182 @@
-/*
-  Manages the remote build configuration, /etc/nix/machines
+/* Manages the remote build configuration, /etc/nix/machines
 
-  See also
-   - ./nix.nix
-   - nixos/modules/services/system/nix-daemon.nix
+   See also
+    - ./nix.nix
+    - nixos/modules/services/system/nix-daemon.nix
 */
 { config, lib, ... }:
 
 let
   inherit (lib)
-    any
-    concatMapStrings
-    concatStringsSep
-    filter
-    getVersion
-    mkIf
-    mkMerge
-    mkOption
-    optional
-    optionalString
-    types
-    versionAtLeast
-    ;
+    any concatMapStrings concatStringsSep filter getVersion mkIf mkMerge
```

Comment:
are you sure that you did run `nixfmt-rfc-style` (https://github.com/NixOS/nixpkgs/blob/26594d3d1f45d6124ba8faee3572f0d358945c94/shell.nix#L34)?
That formatting looks at least not rfc-style'ish to me

Reply:
Looks like I did...